packages/client/src/components/Players/Player.tsx

```diff
@@ -10,12 +10,23 @@ import { Box, BoxProps, Grid } from 'theme-ui';
 type Props = {
   player: PlayerState;
   isActive: boolean;
+  isSpectator: boolean;
   togglePlayerActiveCard: React.Dispatch<React.SetStateAction<string | null>>;
   activePlayerCard: string | null;
 };
 
-export const Player = ({ player, isActive, togglePlayerActiveCard, activePlayerCard, ...rest }: Props & BoxProps) => {
+export const Player = ({
+  player,
+  isActive,
+  isSpectator,
+  togglePlayerActiveCard,
+  activePlayerCard,
+  ...rest
+}: Props & BoxProps) => {
   if (!player) {
```

Comment:
Yeah, it might work... I will try. Also, do you think the `Player` and `Opponent` could be one component? So, it's always a player component, but it is opponents, cards go to the top and are not selectable. They are also a bit inaccurately named at the moment (as an opponent is a player).

Reply:
Go for it but maybe let's merge the other pr first